### ISSUE TITLE
Clean-up of 1.5 references from kraken-lib

### ISSUE THREAD

**Issue by guineveresaenger** [samsung-cnct/kraken-lib#909]
There are still references in the docs, as well as the default config file, to k8s version 1.5, which is no longer supported.

They should be cleaned up. Also, investigate whether current default settings in config file still make sense for >1.5, which may render some ansible tasts unnecessary.

**Comment by philoserf** [comment]
from `kraken generate`d `$HOME/.kraken/config.yml` this snippet remains:

```yaml
   - &rbacKubeAuth
      authz:
        rbac:
          # super_user is required until kubernetes 1.5 is no longer supported by k2.
          # It is not used by kubernetes 1.6 or later.
          super_user: "placeholder"
```

**Comment by philoserf** [comment]
The comment to be removed is located in the files included in this old/unused PR #922

**Comment by oneilcin** [comment]
PR #934
